package/setup.py

```diff
@@ -597,10 +597,10 @@ def long_description(readme):
                        'trajectories generated by CHARMM, Gromacs, NAMD, LAMMPS, or Amber.'),
           long_description=LONG_DESCRIPTION,
           long_description_content_type='text/x-rst',
-          author='Naveen Michaud-Agrawal',
-          author_email='naveen.michaudagrawal@gmail.com',
+          author='MDAnalysis Development Team',
+          author_email='mdanalysis@numfocus.org',
           maintainer='Richard Gowers',
```

Comment:
Is @IAlibay current maintainer or should this still be @richardjgowers ?

Reply:
@richardjgowers still holds all the keys, I'd like to think of myself as the sith apprentice 🙃 

That being said, if we're looking to change it, shouldn't the maintainers be "The MDAnalysis Core Developer Team"?

Reply:
I like your suggestion. At least then blame gets properly apportioned.

@richardjgowers what's your opinion?

Reply:
Just for another data point--SciPy mentions the person holding the release manager position at a given time in the online docs, but not in the source as far as I know: https://scipy.github.io/devdocs/dev/governance/people.html?highlight=tyler%20reddy#release-manager

maintainers is "SciPy Developers"

Reply:
In response to @IAlibay and @richardjgowers comments I am changing the maintainer name to the official title [MDAnalysis Core Developers](https://www.mdanalysis.org/about/#governance). 

(I was striving for grammatical uniformity but If you prefer the article "The" in front I can be swayed.)